actionpack/test/controller/api/params_wrapper_test.rb

```diff
@@ -7,7 +7,7 @@ class UsersController < ActionController::API
     wrap_parameters :person, format: [:json]
 
     def test
-      self.last_parameters = params.except(:controller, :action)
+      self.last_parameters = params.except(:controller, :action).to_unsafe_h
```

Comment:
Yes, but because of #18006 `to_h` only returns _safe_ parameters (after slicing) while `to_unsafe_h` returns everything as-is.

Since this test is testing all parameters and doesn't care about Strong Parameters, `to_unsafe_h` works fine here.